tests/org.eclipse.fordiac.ide.test.ui/META-INF/MANIFEST.MF

```diff
@@ -31,5 +32,5 @@ Require-Bundle: org.eclipse.ui,
  org.eclipse.fordiac.ide.systemmanagement.ui;bundle-version="3.0.0",
  org.eclipse.fordiac.ide.typemanagement;bundle-version="3.0.0",
  org.eclipse.swt,
- org.hamcrest.core,
+ org.hamcrest.library,
```

Comment:
Could you please extract this fix into an own PR. As you can see in other PRs (e.g., ##269 & #274) this is a generic improvement we need and should therefore be separated.